oc-includes/osclass/emails.php

```diff
@@ -706,18 +732,24 @@ function fn_email_new_comment_admin($aItem) {
             osc_item_url(),
             $itemURL
         );
+
         $title_email = osc_apply_filter('email_new_comment_admin_title_after', osc_mailBeauty(osc_apply_filter('email_title', osc_apply_filter('email_new_comment_admin_title', $content['s_title'], $aItem)), $words), $aItem);
         $body_email = osc_apply_filter('email_new_comment_admin_description_after', osc_mailBeauty(osc_apply_filter('email_description', osc_apply_filter('email_new_comment_admin_description', $content['s_text'], $aItem)), $words), $aItem);
 
-        $emailParams = array(
-            'from'      => osc_contact_email(),
-            'to'        => $admin_email,
-            'to_name'   => __('Admin mail system'),
-            'subject'   => $title_email,
-            'body'      => $body_email,
-            'alt_body'  => $body_email
-        );
-        osc_sendMail($emailParams);
+        $admins = Admin::newInstance()->listAll();
+        foreach($admins as $admin) {
```

Comment:
why loop the sendMail function instead of using an array in the "to" param (or bcc or cc)?